### ISSUE TITLE
Superflous dependencies declared

### ISSUE THREAD

**Issue by sjsf** [Jsondb/jsondb-core#1]
The following libraries are defined as compile dependencies:

```
          "org.slf4j:jcl-over-slf4j:1.7.21",
          "org.apache.logging.log4j:log4j-core:2.6.2",
          "org.apache.logging.log4j:log4j-api:2.6.2",
          "org.apache.logging.log4j:log4j-slf4j-impl:2.6.2",
```

However, as far as I can see this project uses the slf4j api, so it should be up to the application that uses jsondb to decide and package the desired logging implementation. I therefore suggest to drop them from here - or maybe add them as testCompile dependency if really needed for running the tests.

**Comment by FarooqKhan** [comment]
Thanks for fixing this Simon
Farooq

**Comment by FarooqKhan** [comment]
This was released to Maven as build 1.0.64

MavenCentral should be updated in a hour,
mvnrepository.com should start showing it in about 5 to 7 days
